exchanges/gateio/gateio_websocket.go

```diff
@@ -546,29 +540,23 @@ func (e *Exchange) processMarginBalances(ctx context.Context, data []byte) error
 		Event   string            `json:"event"`
 		Result  []WsMarginBalance `json:"result"`
 	}{}
-	err := json.Unmarshal(data, &resp)
-	if err != nil {
-		return err
-	}
-	creds, err := e.GetCredentials(ctx)
-	if err != nil {
+	if err := json.Unmarshal(data, &resp); err != nil {
 		return err
 	}
-	changes := make([]account.Change, len(resp.Result))
+	subAccts := accounts.SubAccounts{accounts.NewSubAccount(asset.Margin, "")}
```

Comment:
Why not use user like spot?

Reply:
🤷 The code that was there before didn't use it. 
This seems right, though:

Fixed https://github.com/gbjk/gocryptotrader/commit/61f500242

> Why not use user like spot?

This took so many attempts to read 😂